redbot/cogs/economy/economy.py

```diff
@@ -252,6 +252,27 @@ def __init__(self, bot: Red):
                         credits_name
                     )
                 )
+                await ctx.send(
+                    _("You currently have {} {}.".format(
```

Comment:
The i18n object requires  `_("...").format()`

Reply:
whoops, I'll fix that now